maintenance/recipes/turn_on.rb

```diff
@@ -0,0 +1,6 @@
+bash 'turn maintenance on' do
+  estimated_end_time = node[:maintenance][:estimated_end_time]
+  sed_command = "s/<span id=\"estimated-end-time\">.*</span>/#{estimated_end_time}/"
+  code "sudo -S -u deploy sed -i \"#{sed_command}\" /srv/www/jiffyshirts/current/public/maintenance.html"
+  code 'sudo -S -u deploy touch /srv/www/jiffyshirts/current/public/offline'
```

Comment:
Wouldn't second `code` override the first one?

Reply:
I think answer is yes, but I could be wrong. If I'm right, there's a `touch` action for `file` resource:
```ruby
file '/srv/www/jiffyshirts/current/public/offline' do
  user 'deploy'
  action :touch
end
```

Reply:
Updated in a chef way